editor/sidebar/post-status/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import { Component } from 'element';
+import PanelBody from 'components/panel-body';
+import PanelBodyToggle from 'components/panel-body-toggle';
+import FormToggle from 'components/form-toggle';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+
+class PostStatus extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			opened: true,
+		};
+		this.toggle = this.toggle.bind( this );
+	}
+
+	toggle( event ) {
+		event.preventDefault();
+		this.setState( {
+			opened: ! this.state.opened,
+		} );
+	}
+
+	render() {
+		return (
+			<PanelBody>
+				<PanelBodyToggle
```

Comment:
Sure, that seems fine. It's worked for us elsewhere as a generic Accordion component with `initialExpanded`, `title` (and per previous note, `subtitle`).

https://github.com/Automattic/wp-calypso/blob/f19b119/client/components/accordion/index.jsx#L16-L22